src/debug/debugSession.ts

```diff
@@ -67,7 +73,10 @@ export class DebugSession implements IDebugSession {
         const containerEnv = {};
         const portInfo = await this.debugProvider.resolvePortsFromFile(dockerfile, containerEnv);
         if (!portInfo || !portInfo.debugPort || !portInfo.appPort) {
-            vscode.window.showErrorMessage("Cannot resolve debug/application port from Dockerfile.");
+            const answer = await vscode.window.showErrorMessage("Cannot resolve debug/application port from Dockerfile, please refer to the wiki for the usage.", "Open in Browser");

```

Comment:
Similar comments e.g. `Cannot resolve debug/application port from Dockerfile. See the documentation for how to use this command.`